sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -1963,3 +1963,10 @@ func (o *oomSniffer) Scan(r io.Reader) {
 		}
 	}()
 }
+
+func (host *nodeLanguageHost) Link(
+	ctx context.Context, req *pulumirpc.LinkRequest,
+) (*pulumirpc.LinkResponse, error) {
+	// The NodeJS language host does not support linking, so we return an empty response.
+	return &pulumirpc.LinkResponse{}, nil
```

Comment:
Should it not be unimplemented instead? Same for any other language hosts we haven't implemented yet.